web/client/components/catalog/RecordItem.jsx

```diff
@@ -211,7 +213,7 @@ class RecordItem extends React.Component {
     };
 
     addLayer = (wms) => {
-        const {url, params} = removeParameters(wms.url, ["request", "layer", "service", "version"]);
+        const {url, params} = removeParameters(ConfigUtils.normalizeUrl(wms.url), ["request", "layer", "service", "version"].concat(this.props.authkeyParamNames));
```

Comment:
This `removeParameters` seems to be a copy of the one in ConfigUtils, isn't it?
If it is, we should remove this local definition.

Reply:
this one returns an object containing:
- a first part of the url till the querystring
- the params not removed